server/cache/two_queue.go

```diff
@@ -91,6 +91,7 @@ func (c *TwoQueue) Put(key uint64, value interface{}) {
 	c.recent.Put(key, value)
 }
 
+// revive:disable-next-line:flag-parameter
```

Comment:
Is there a better way to fix this warning?

Reply:
It's possible to just remove this in the linter settings if you don't like this rule. But otherwise this is a good solution for now. I think its better to add the comment here then to continually see the output in the linter.

Reply:
I think the rule is useful - but not always useful. The method here is not exported and is only used in the `Put` method not far above. I suppose no need to be dogmatic here.